lib/sles4sap_publiccloud.pm

```diff
@@ -363,9 +363,13 @@ sub check_takeover {
 
   TAKEOVER_LOOP: while (1) {
         my $topology = $self->get_hana_topology();
+        record_info("Cluster topology", $topology);
```

Comment:
how printing `$topology` is useful? Is it not just a reference (address) to a structure?

Reply:
just to double check what type is and see what kind of content has

Reply:
but you will not see it, you will just see an address

Reply:
removed